src/test/java/duke/command/ParserTest.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import org.junit.jupiter.api.Test;
+
+import duke.exception.DukeParserException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ParserTest {
+    @Test
+    public void parseDate_validDateFormat_success() throws Exception {
```

Comment:
if i am not wrong, it is not recomended to have methods with '_' (it's against coding standards i think)

Reply:
Haha, perhaps you might wanna look at [https://nus-cs2103-ay1920s1.github.io/website/schedule/week3/topics.html#tools-junit-junit-basic](url)